packages/azure-storage/CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # @openfn/language-azure-storage
 
+## 2.0.13 - 17 April 2025
+
+### Patch Changes
+
+* Add query parameters to URL request logs
```

Comment:
This doesn't look right? Azure doesn't use common request?

Reply:
If you check the changelogs, we have a dependency update for common, `adaptors.js` is also exporting common, and we have this `"@openfn/language-common": "workspace:*"` in its package.json

Reply:
Yes, `azure-storage` depends on common, but it is not affected by this change because it does not use the `request`  function. So the changelogs are incorrect. You won't suddenly start seeing query parameters  getting logged out in azure-storage

Changeset will automatically detect the dependency change  and generate a log for us. That's great, but it means that changelogs start looking like this:

![image](https://github.com/user-attachments/assets/c12f5c54-be35-437a-bbaf-2a48bc40cd66)

The ask here was to calculate which adaptors are _actually_ affected and generate a nice, accurate changelog fo them.

I've had to do that for another PR this morning so I'm going to hijack that changeset and add this note. I'll merge it all together now.